### PR TITLE
interactive bots: Run the helloworld bot for Smehnov

### PR DESCRIPTION
**Pull request by Smehnov**
Task: https://codein.withgoogle.com/dashboard/task-instances/4676513926479872/

### CODE REVIEW

**Comment by jajodiaraghav** [comment]
Hi @Smehnov You did awesome work!
All you have to do now is squash your commits.  This might help: https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

Head over to https://chat.zulip.org/#narrow/stream/GCI.20help in case you are facing any trouble.
Thanks

**Comment by abhijeetkaur** [comment]
Great work @Smehnov. You seem to have a good knowledge of git. :) 
Could you please take a larger screenshot so that it shows that the bot is replying in #Verona stream.
>messages sent by you to the bot in the stream "Verona"

This is a requirement for the submission of the task
Once you submit a greater screenshot image showing the stream name Verona (on the left side bar), I'll approve your work. Thanks!

File: interactive-bots/helloworld/Smehnov/notes.md [hunk 1]
```diff
@@ -0,0 +1 @@
+I had many problems while I did this task, but I solved them thanks to mentors
```
Comment (abhijeetkaur) [inline interactive-bots/helloworld/Smehnov/notes.md]: * I suggest you to end the sentence with a period (.). 
* You could be even more descriptive about your problems and how you solved them. This is not necessary though! :)

**Comment by Smehnov** [comment]
@abhijeetkaur I did it

**Comment by eeshangarg** [comment]
@Smehnov: Great work! Merged and approved! :)

**Pull request merged**